_sass/components/_card.scss

```diff
@@ -0,0 +1,158 @@
+.large-card {
+  border-radius: 16px;
+  overflow: hidden;
+  background-color: $white-offset;
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  width: 360px;
+  height: 496px;
+  left: 16px;
+  top: 16px;
+  align-items: flex-start;
+  padding: 0px;
```

Comment:
Can we extract rules that are common across `.large-card` and `.small-card` into a `.card`?

Reply:
Yeah just extracted common rules